### DISCUSSION TITLE
Include City Name Column in 'All Task' Downloads

### DISCUSSION THREAD

**Discussion by devselina** [omkarcloud/google-maps-scraper#180] [Ideas]
You have done some magic here. I have been using this for the last few weeks and am grateful to have stumbled upon your great work. Thank you.

Suggested Addition (Pro or Free)

For the "All Task" download, please include a column in the download file that contains the corresponding location or city name, especially when using the 'Extract Cities By Country' feature. For example, if the search query is 'restaurants' and I choose 'Extract Cities By Country - United Kingdom (823 cities)', every restaurant in the download file should have a column listing the city.

I know one can use the address to extract the city name, but sometimes there is a mismatch with the extracted cities. I would rather work with the listed cities directly.

**Comment by Chetan11-dev** [comment]
Will release by July 20th

**Reply by devselina** [reply]
Great, thank you.

**Reply by Chetan11-dev** [reply]
release, update following https://github.com/omkarcloud/google-maps-scraper/blob/master/advanced.md#-how-to-update-the-scraper-to-the-latest-version
